### PR TITLE
MODKBEKBJ-378 add number of records to GET endpoints

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Return number of records used by each Access Type ([JIRA](https://issues.folio.org/browse/MODKBEKBJ-378))

## Approach
* Update raml
* Update GET endpoints to return number of records used by each Access Type
* Update tests

### CODE REVIEW

File: ramls/types/accessTypes/accessTypeCollectionItem.json [hunk 1]
```diff
@@ -35,6 +35,11 @@
       "$ref": "userDisplayInfo.json",
       "readonly": true
     },
+    "recordUsage": {
```
Comment (dmtkachenko) [inline ramls/types/accessTypes/accessTypeCollectionItem.json]: Could you rename it to usageNumber

File: src/main/java/org/folio/service/accesstypes/AccessTypesServiceImpl.java [hunk 2]
```diff
@@ -59,12 +62,15 @@
   @Override
   public CompletableFuture<AccessTypeCollection> findAll(Map<String, String> okapiHeaders) {
     return repository.findAll(tenantId(okapiHeaders))
+      .thenCombine(mappingService.findAll(okapiHeaders), this::setEachRecordUsage)
```
Comment (dmtkachenko) [inline src/main/java/org/folio/service/accesstypes/AccessTypesServiceImpl.java]: This is not the best way of getting number of usages. When you have thousand of mappings the performance is going down significantly.
You need to perform the calculation on DB side by grouping records and counting the totals

File: ramls/types/accessTypes/accessTypeCollectionItem.json (hunk 1 above)
Comment (psmagin) [reply ramls/types/accessTypes/accessTypeCollectionItem.json]: Renamed

File: src/main/java/org/folio/service/accesstypes/AccessTypesServiceImpl.java (hunk 2 above)
Comment (psmagin) [reply src/main/java/org/folio/service/accesstypes/AccessTypesServiceImpl.java]: Fixed

**Pull request merged**